test-jest/chart/AreaChart.spec.tsx

```diff
@@ -131,6 +164,8 @@ describe('AreaChart', () => {
     expect(container.querySelectorAll('.recharts-area')).toHaveLength(0);
   });
 
+  // TODO: add more tests
```

Comment:
Please either remove the todo, or explicate, i.e. write down what tests should be added. 
A single TODO only saying that we should add more tests would be valid anywhere and does not provide context / direction.

Reply:
These TODO is to mind myself actually, I'm going to add it today or tomorrow. I will request for review once I've done, sorry for the ambiguity